File: Python/백준/Bronze/9950. Rectangles/Rectangles.py
while(True):
    a,b,c = map(int,input().split())
    if a==b and b==c and a==0:
        break
    if a==0:
        print(c//b,b,c)
    elif b==0:
        print(a,c//a,c)
    else:
        print(a,b,a*b)

File: C#/백준/Bronze/9950. Rectangles/Rectangles.cs
using System;

namespace q{
    class w{
        static void Main(){
            while(true){
                int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
                int a = aa[0]; int b = aa[1]; int c = aa[2];
                if(a==0 && b==0 && c==0)
                    break;
                else if(a==0)
                    Console.WriteLine($"{c/b} {b} {c}");
                else if(b==0)
                    Console.WriteLine($"{a} {c/a} {c}");
                else
                    Console.WriteLine($"{a} {b} {a*b}");
            }
        }
    }
}